src/scss/base/_mixins.scss

```diff
@@ -10,10 +10,49 @@
 @mixin setBgColorAndHover($baseColor) {
   background-color: $baseColor;
 
-  /* we use :not(.btn--disabled) to prevent hover and active states for disabled btn */
+  // we use :not(.btn--disabled) to prevent hover and active states for disabled btn
   &:not(.btn--disabled):active,
   &:not(.btn--disabled):hover {
     background-color: darken($baseColor, 10%);
   }
 }
 
+//media query mixin
+@mixin tablet {
+  @media (min-width: #{$mobile}) {
+    @content;
+  }
+}
+
+@mixin desktop {
+  @media (min-width: #{tablet}) {
+    @content;
+  }
+}
+
+@mixin large-desktop {
+  @media (min-width: #{desktop}) {

```

Comment:
there is missing $ near desktop and this mixins are already implement - please be aware of merge. It will be good to merge master into your branch